pages/search.js

```diff
@@ -136,7 +137,7 @@ export default function Search({ data }) {
       <Page>
         <h1 className="text-4xl mb-4 font-bold">Search</h1>
 
-        <div className="flex flex-wrap justify-center space-x-3 mb-4">
+        <button className="flex flex-wrap justify-center space-x-3 mb-4">
```

Comment:
This is just 1 button? Each tag needs to be its own button

Reply:
The tags are being mapped over and rendered as Tag components with button elements, If there are 10 tags available the code will render only the available tags as buttons, so we can have 10 buttons, same logic as div with replace of div we are using buttons now.

Reply:
@kishoreravi24 this div is just a wrapper . The `div` inside the `Tag` component needs to be changed to a button. This needs to remain a `div`

Reply:
@ChinmayMhatre thanks for the explanation, done fixed the cc comments please check now 👍